src/containers/main/Dashboard/components/BlockExplorerMini/useBlocks.ts

```diff
@@ -0,0 +1,64 @@
+import { useQuery } from '@tanstack/react-query';
+
+export const BLOCKS_KEY = ['blocks'];
+const address = 'https://textexplore.tari.com';
```

Comment:
It's this url would need to be swapped out on a per network basis.

For mainnet this is the right address. For nextnet and esme it needs to be:

Emse: https://textexplore-esmeralda.tari.com/
Nextnet: https://textexplore-nextnet.tari.com/